index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navmenu">
+      <h3>My Logo</h3>
+    </nav>
+    <img src="image/Union.png" alt="" />
+
```

Comment:
- [ OPTIONAL]  Your Humburg menu should be at the right side of you page. Please fix, you can use flex box to style it . 
![image](https://user-images.githubusercontent.com/43877689/178114939-de16e0c5-caab-42fe-8099-38af08e945c0.png)